### ISSUE TITLE
Second draft of topic definitions 

### ISSUE THREAD

**Issue by nniiicc** [PugetSoundClinic-PIT/OSH_Review#13]
- When you're ready assign to @nniiicc for review

**Comment by sarah114tran** [comment]
@nniiicc I am ready for review! 

https://docs.google.com/document/d/1BTqMdEd-tYHfFPzMcjk83NBF4Xl8qs6IExTVdR8U9R8/edit?usp=sharing
* Whats in bold is the original version, and the ones below are our versions. 

Sarah next steps: 

- [x] Sample another set of ten and try to apply the new definitions
- [x] See if we need to look at the entire abstract to determine topic more confidently

**Comment by sarah114tran** [comment]
Update: coded a new set of 10, and defining the topics did go a bit easier with the new definitions. Also, I have found that referring back to the abstracts of the paper may help resolve differences.

**Comment by nniiicc** [comment]
@sarah114tran -these are excellent. Thank you for the careful eye on their edits. Here is what I did:
- I copied the same 10 and coded with your defintions. 
- We agreed on all but one "Can Open Source Hardware Mechanical Ventilator (OSH-MVs) initiatives help cope with the COVID-19 health crisis? Taxonomy and state of the art" - I coded `Characteristcs` and you coded `Development` 
- After reading the abstract I agree that it should be `Development`

Here are my proposed next steps: 
- Sarah and Lindsey code the rest of the papers (in total) 
- When you are finished - find a time to meet and resolve any differences.

A few other ideas:
- I will work on the classifier so we can report the distribution of papers (empirical, review, etc) 
- I think we should NOT include any review articles in our scoping review even if they are systematic and empirical. Instead we should include a separate section that discusses previous review articles and describe some of their major findings - I can write out the logic for this in the paper draft. If I can write the classifier correctly we can identify review articles automatically. If not we can probably just do some keyword searching titles (which I can do) 
- In the paper that we developed the topical categories from ( https://dl.acm.org/doi/pdf/10.1109/FLOSS.2009.5071355) they cite another paper that they used to structure their presentation of their systematic review results ([doi: 10.1109/APSEC.2008.50](https://doi.org/10.1109/APSEC.2008.50)) -essentially calling things "facts, trends, and directions"  - I think this is OVERLY reductive - but it might help spark some ideas for how we divide up the results section
